lvsfunc/misc.py

```diff
@@ -408,6 +410,53 @@ def upscale_chroma(n: int, f: vs.VideoFrame) -> vs.VideoNode:
     return cast(F, inner)
 
 
+def colored_clips(amount: int, max_hue: int = 300, **kwargs: Any) -> List[vs.VideoNode]:
+    """
+    Returns a list of BlankClips with randomized unique colors. Useful maybe for comparison functions
+    or just for getting multiple uniquely colored BlankClips for testing purposes.
+
+    `max_hue` is the maximum hue that will be returned from the HSL color model.
+
+    Written by Dave <orangechannel@pm.me>.
+
+    :param amount:  Number of `vapoursynth.VideoNode`s to return
```

Comment:
you probably want \`\`vapoursynth.VideoNode\`\` for code tags, single backticks will render as italics